### ISSUE TITLE
only in Giardia?  GO:0097593    ventral disc microtubule array

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#16102]
not sure about this one:
GO:0097593    ventral disc microtubule array

never in fungi, or better

**Comment by hdrabkin** [comment]
The ventral disc term already has a taxon constraint; can you not see in quickgo (ie, part_of inheritance)? 
@cmungall : isn't this suppose to show up in quickGO?

**Comment by cmungall** [comment]
A bunch of ventral disc 

```
[Term]
id: GO:0097591 ! ventral disc lateral crest
name: ventral disc lateral crest
is_a: GO:0044422  ! organelle part

[Term]
id: GO:0097592 ! ventral disc overlap zone
name: ventral disc overlap zone
is_a: GO:0044422  ! organelle part
relationship: has_part GO:0097593 ! ventral disc microtubule array

[Term]
id: GO:0097593 ! ventral disc microtubule array
name: ventral disc microtubule array
is_a: GO:0044422  ! organelle part
relationship: has_part GO:0005874 ! microtubule

[Term]
id: GO:0097594 ! ventral disc dorsal microribbon
name: ventral disc dorsal microribbon
is_a: GO:0044422  ! organelle part

[Term]
id: GO:0097595 ! ventral disc crossbridge
name: ventral disc crossbridge
is_a: GO:0044422  ! organelle part
```

**Comment by hdrabkin** [comment]
seems these should all be part of vd; I can do this but would like @krchristie  to comment

**Comment by paolaroncaglia** [comment]
@hdrabkin
I went back to the notes I exchanged with the Giardia experts at the time, and I think we can safely assume that the 5 terms listed by @cmungall above can be made part_of ‘ventral disc’. I added those terms back in 2014 so can’t quite recall why I didn’t do that myself. My best bet is that at the time we were a bit cautious with adding reciprocal links? Or perhaps I focused on the has_parts too hard ;-)
To confirm, if you add the part_of links, the taxon constrain that’s already in place for ‘ventral disc’ will address @ValWood’s initial request.
Thanks,
Paola
